src/back-end/lib/resources/user.ts

```diff
@@ -158,17 +160,12 @@ const resource: Resource = {
             if (!permissions.updateUser(request.session, request.params.id)) {
               return invalid({ permissions: [permissions.ERROR_MESSAGE] });
             }
-            const { capabilities } = request.body.value;
-            const validatedCapabilities = userValidation.validateCapabilities(capabilities);
+            const validatedCapabilities = userValidation.validateCapabilities(request.body.value);
             if (isValid(validatedCapabilities)) {
-              return valid(adt('updateCapabilities' as const, {
-                capabilities: validatedCapabilities.value
-              }));
+              return valid(adt('updateCapabilities' as const, validatedCapabilities.value));
             } else {
               return invalid({
-                user: adt('updateCapabilities' as const, {
-                  capabilities: getInvalidValue(validatedCapabilities, undefined)
-                })
+                user: adt('updateCapabilities' as const, getInvalidValue(validatedCapabilities, undefined) as string[][])
```

Comment:
2nd arg to `getInvalidValue` should be an empty array, not undefined per the shared type.

Reply:
Thanks, corrected.